[2025-02-25T09:49:56.164+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-25T09:49:56.237+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-25T09:49:56.240+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-25T09:49:56.308+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): store_raw_data_in_hdfs> on 2025-01-20 00:00:00+00:00
[2025-02-25T09:49:56.323+0000] {standard_task_runner.py:57} INFO - Started process 760 to run task
[2025-02-25T09:49:56.329+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'yfinance_ingestion_dag', 'store_raw_data_in_hdfs', 'scheduled__2025-01-20T00:00:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/test_hello.py', '--cfg-path', '/tmp/tmpn5dsepbm']
[2025-02-25T09:49:56.334+0000] {standard_task_runner.py:85} INFO - Job 82: Subtask store_raw_data_in_hdfs
[2025-02-25T09:49:56.544+0000] {task_command.py:415} INFO - Running <TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [running]> on host 6dbd75e48aaa
[2025-02-25T09:49:56.864+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etudiant' AIRFLOW_CTX_DAG_ID='yfinance_ingestion_dag' AIRFLOW_CTX_TASK_ID='store_raw_data_in_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-20T00:00:00+00:00'
[2025-02-25T09:50:37.111+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-25T09:50:37.153+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=yfinance_ingestion_dag, task_id=store_raw_data_in_hdfs, execution_date=20250120T000000, start_date=20250225T094956, end_date=20250225T095037
[2025-02-25T09:50:37.221+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-25T09:50:37.295+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-26T20:25:24.418+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-26T20:25:24.512+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-26T20:25:24.527+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-26T20:25:23.800+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): store_raw_data_in_hdfs> on 2025-01-20 00:00:00+00:00
[2025-02-26T20:25:23.820+0000] {standard_task_runner.py:57} INFO - Started process 1248 to run task
[2025-02-26T20:25:23.850+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'yfinance_ingestion_dag', 'store_raw_data_in_hdfs', 'scheduled__2025-01-20T00:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpem3x4_o5']
[2025-02-26T20:25:23.871+0000] {standard_task_runner.py:85} INFO - Job 138: Subtask store_raw_data_in_hdfs
[2025-02-26T20:25:24.202+0000] {task_command.py:415} INFO - Running <TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [running]> on host c96a1011f5a3
[2025-02-26T20:25:24.633+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etudiant' AIRFLOW_CTX_DAG_ID='yfinance_ingestion_dag' AIRFLOW_CTX_TASK_ID='store_raw_data_in_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-20T00:00:00+00:00'
[2025-02-26T20:25:59.459+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-26T20:25:59.503+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=yfinance_ingestion_dag, task_id=store_raw_data_in_hdfs, execution_date=20250120T000000, start_date=20250226T202524, end_date=20250226T202559
[2025-02-26T20:25:59.635+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-26T20:25:59.749+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-26T20:39:13.818+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-26T20:39:13.868+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-26T20:39:13.869+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-26T20:39:13.925+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): store_raw_data_in_hdfs> on 2025-01-20 00:00:00+00:00
[2025-02-26T20:39:13.943+0000] {standard_task_runner.py:57} INFO - Started process 521 to run task
[2025-02-26T20:39:13.954+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'yfinance_ingestion_dag', 'store_raw_data_in_hdfs', 'scheduled__2025-01-20T00:00:00+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpwr6hustx']
[2025-02-26T20:39:13.961+0000] {standard_task_runner.py:85} INFO - Job 239: Subtask store_raw_data_in_hdfs
[2025-02-26T20:39:14.216+0000] {task_command.py:415} INFO - Running <TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [running]> on host 3c0faf52b292
[2025-02-26T20:39:14.694+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etudiant' AIRFLOW_CTX_DAG_ID='yfinance_ingestion_dag' AIRFLOW_CTX_TASK_ID='store_raw_data_in_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-20T00:00:00+00:00'
[2025-02-26T20:39:14.174+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-26T20:39:14.219+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=yfinance_ingestion_dag, task_id=store_raw_data_in_hdfs, execution_date=20250120T000000, start_date=20250226T203913, end_date=20250226T203914
[2025-02-26T20:39:14.389+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-26T20:39:14.788+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-26T20:41:54.797+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-26T20:41:54.823+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-26T20:41:54.824+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-26T20:41:54.864+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): store_raw_data_in_hdfs> on 2025-01-20 00:00:00+00:00
[2025-02-26T20:41:54.876+0000] {standard_task_runner.py:57} INFO - Started process 1697 to run task
[2025-02-26T20:41:54.882+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'yfinance_ingestion_dag', 'store_raw_data_in_hdfs', 'scheduled__2025-01-20T00:00:00+00:00', '--job-id', '319', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpotjba3qt']
[2025-02-26T20:41:54.887+0000] {standard_task_runner.py:85} INFO - Job 319: Subtask store_raw_data_in_hdfs
[2025-02-26T20:41:54.995+0000] {task_command.py:415} INFO - Running <TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [running]> on host 3c0faf52b292
[2025-02-26T20:41:55.286+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etudiant' AIRFLOW_CTX_DAG_ID='yfinance_ingestion_dag' AIRFLOW_CTX_TASK_ID='store_raw_data_in_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-20T00:00:00+00:00'
[2025-02-26T20:42:03.812+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-26T20:42:03.832+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=yfinance_ingestion_dag, task_id=store_raw_data_in_hdfs, execution_date=20250120T000000, start_date=20250226T204154, end_date=20250226T204203
[2025-02-26T20:42:03.976+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-26T20:42:04.025+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-27T14:55:37.156+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-27T14:55:37.188+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [queued]>
[2025-02-27T14:55:37.189+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-02-27T14:55:37.234+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): store_raw_data_in_hdfs> on 2025-01-20 00:00:00+00:00
[2025-02-27T14:55:37.246+0000] {standard_task_runner.py:57} INFO - Started process 351 to run task
[2025-02-27T14:55:37.259+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'yfinance_ingestion_dag', 'store_raw_data_in_hdfs', 'scheduled__2025-01-20T00:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpandygosi']
[2025-02-27T14:55:37.269+0000] {standard_task_runner.py:85} INFO - Job 54: Subtask store_raw_data_in_hdfs
[2025-02-27T14:55:37.552+0000] {task_command.py:415} INFO - Running <TaskInstance: yfinance_ingestion_dag.store_raw_data_in_hdfs scheduled__2025-01-20T00:00:00+00:00 [running]> on host a71f042480b6
[2025-02-27T14:55:38.040+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='etudiant' AIRFLOW_CTX_DAG_ID='yfinance_ingestion_dag' AIRFLOW_CTX_TASK_ID='store_raw_data_in_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-20T00:00:00+00:00'
[2025-02-27T14:55:38.098+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/pipeline.py", line 82, in store_raw_data_in_hdfs
    with open(local_file, 'w') as f:
PermissionError: [Errno 13] Permission denied: '/mnt/hadoop_data/yfinance_raw.json'
[2025-02-27T14:55:38.163+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=yfinance_ingestion_dag, task_id=store_raw_data_in_hdfs, execution_date=20250120T000000, start_date=20250227T145537, end_date=20250227T145538
[2025-02-27T14:55:38.220+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 54 for task store_raw_data_in_hdfs ([Errno 13] Permission denied: '/mnt/hadoop_data/yfinance_raw.json'; 351)
[2025-02-27T14:55:38.268+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-02-27T14:55:38.389+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
